### ISSUE TITLE
Error while re-running `SFDX: Start Function`

### ISSUE THREAD

**Issue by joshwlewis** [forcedotcom/salesforcedx-vscode#3677]
### Summary

Error while running `SFDX: Start Function`

### Steps To Reproduce:

1.  Run `SFDX: Start Function`
2. Run `SFDX: Stop Function`
3. Run `SFDX: Start Function`

### Expected result

It should be possible to stop the function and restart it.

### Actual result

```
Stopping Function
14:15:52.86 Starting SFDX: Start Function

ENOENT: no such file or directory, lstat 'extension-output-salesforce.salesforcedx-vscode-core-#1'
14:16:13.432 Starting SFDX: Start Function

ENOENT: no such file or directory, lstat 'extension-output-salesforce.salesforcedx-vscode-core-#1'
```

### Additional information

**Salesforce Extension Version in VS Code**:

Salesforce Extension Pack v53.4.2

**SFDX CLI Version**:
sfdx-cli/7.122.1 darwin-x64 node-v14.18.0

**OS and version**:

Mac OS Big Sur 11.6.1

**Comment by jeffb-sfdc** [comment]
I was able to reproduce this issue, though, it doesn't appear to be an issue with stopping, it appears to be an issue with starting in the first place.  When I ran `SFDX: Start Function`, I got:

```
16:28:34.361 Starting SFDX: Start Function

ENOENT: no such file or directory, lstat 'extension-output-salesforce.salesforcedx-vscode-core-#1'
```

...which I don't think is expected.

**Comment by gbockus-sf** [comment]
This issue was fixed by W-10257676 / https://github.com/forcedotcom/salesforcedx-vscode/pull/3802
